pkg/apis/servicecatalog/types.go

```diff
@@ -145,10 +145,15 @@ type CommonServiceBrokerSpec struct {
 // ServiceClass allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
 //   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
```

Comment:
I know that you didn't write this, but the key for this property should be spec.externalName.